src/library/Text/types.js

```diff
@@ -13,6 +13,14 @@ type Appearance = $Keys<typeof APPEARANCE>;
 type FontWeight = $Keys<typeof FONT_WEIGHT> | number;
 
 export type TextProps = {
+  /* TargetX Custom Props */
+  altFont?: string,
```

Comment:
@tx-trent @tx-michael 

Poll: What do you think of this prop name `altFont`?

Its purpose is to support a secondary/alternative font family. e.g., Proxima Nova (as Primary) and Proxima Nova Semi Bold (Secondary/Alternative).

Other ones I've considered:
- `<Text alt .../>` (by itself)
- `<Text altFamily .../>`
- `<Text secondary .../>`

Thoughts?

Reply:
I would go with `secondary` here.